.github/workflows/build-and-test-all-linux.yml

```diff
@@ -55,4 +55,7 @@ jobs:
         hungarian_signing_key: ${{secrets.hungarian_signing_key}}
         hungarian_tax_payer_id: ${{secrets.hungarian_tax_payer_id}}
         hungarian_encryption_key: ${{secrets.hungarian_encryption_key}}
+
+        italian_username: ${{secrets.italian_username}}
+        italian_password: ${{secrets.italian_password}}
```

Comment:
How come the builds are green when I didn't  put the secrets there yet? Did somebody else do it?

Reply:
@marektresnak did.